tests/apps/scout/test_scout_models.py

```diff
@@ -84,7 +89,56 @@ def test_convert_other_sex(other_sex_case_output: str):
     assert case["individuals"][0][Pedigree.SEX] == Gender.OTHER
 
     # WHEN validating the output with model
-    case_obj = ScoutExportCase(**case)
+    case_obj = ScoutExportCase.model_validate(case)
 
     # THEN assert that the sex has been converted to "0"
-    assert case_obj.dict()["individuals"][0][Pedigree.SEX] == PlinkGender.UNKNOWN
+    assert case_obj.model_dump()["individuals"][0][Pedigree.SEX] == PlinkGender.UNKNOWN
+
+
+def test_set_parent_when_provided():
+    """Test to validate that the parent value is not altered if a string is provided."""
+
+    # GIVEN a valid parent
+    father: str = "father"
```

Comment:
There are constants for these pedigree terms

Reply:
Fixed!